cclib/parser/orcaparser.py

```diff
@@ -1429,17 +1433,40 @@ def energy_intensity(line):
         #    6:      2069.36    1.674341  ( -0.000000   0.914970  -0.914970)
         #    7:      3978.81   76.856228  (  0.000000   6.199041  -6.199042)
         #    8:      4113.34   61.077784  ( -0.000000   5.526201   5.526200)
+        # ...
+
+        # ORCA 5 example
+        # -----------
+        # IR SPECTRUM
+        # -----------
+        #
+        # Mode   freq       eps      Int      T**2         TX        TY        TZ
+        #       cm**-1   L/(mol*cm) km/mol    a.u.
+        # ----------------------------------------------------------------------------
+        #  6:     45.66   0.000006    0.03  0.000039  ( 0.000000  0.000000  0.006256)
+        #  7:     78.63   0.000000    0.00  0.000000  ( 0.000000  0.000000  0.000000)
+        # ...
         if line[:11] == "IR SPECTRUM":
-            self.skip_lines(inputfile, ['d', 'b', 'header', 'd'])
+            if self.metadata['package_version'][0] == '4':
+                self.skip_lines(inputfile, ['d', 'b', 'header', 'd'])
+                regex = r'\s+(?P<num>\d+):\s+(?P<frequency>\d+\.\d+)\s+(?P<intensity>\d+\.\d+)'
+            elif self.metadata['package_version'][0] == '5':
+                self.skip_lines(inputfile, ['d', 'b', 'header', 'units', 'd'])
+                regex = r'\s+(?P<num>\d+):\s+(?P<frequency>\d+\.\d+)\s+(?P<eps>\d+\.\d+)\s+(?P<intensity>\d+\.\d+)'
+            else:
+                raise Exception('Unsupported version: ' + self.metadata['package_version'])
```

Comment:
Is there a better approach besides checking for 4 vs 5 in version numbers?

Reply:
Nope, this is actually one of the advantages of having the package version available IMO.

Reply:
It's good that package version was available, but if there were (many) more than just these two cases, it seems like another approach would be better.

Reply:
if package version is not parsed (if future versions change the format) then this would result in a KeyError right? Can we check if `package_version` is present and if not just take the newer regex.